schemas/yaml/BibliographicReference.yml

```
@@ -192,6 +204,7 @@ examples:
     doi:             '10.1515/lingty-2013-0013'
     id:              '4b735346-d174-456d-9945-1b00d5ed776b'
     issue:           '2'
+    link:            'http://www.TESTURL.com'
```

```suggestion
    link:            'https://benjamins.com/catalog/cal.20.02hie'
```

This is the actual URL for this reference. Again just to make this example data more realistic.